SEOImages: User Story 1463880, Part 30 (#11410)
dotnet/docs: docs/framework/wpf/app-development/navigation-topologies-overview.md

  
 This topic covers three common navigation topologies: *fixed linear*, *fixed hierarchical*, and *dynamically generated*. Each navigation topology is demonstrated with a sample that has a [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)] like the one that is shown in the following figure:  
  

>  ![Screenshot that shows task pages with data items.](./media/navigation-topologies-overview/navigation-topology-data-items.png)  

Reviewer: ```suggestion
 ![Task pages with data items and navigation buttons.](./media/navigation-topologies-overview/navigation-topology-data-items.png)  
```
